### ISSUE TITLE
Foundations: Incorrect design image being used for landing page project 

### ISSUE THREAD

**Issue by Eduardo06sp** [TheOdinProject/curriculum#28702]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

The assignment provides 2 design files, one of which shows you all the styles used for the landing page example:
> Download the design images and take a look at what you’re going to be creating here. [Image One (Full Design)](https://cdn.statically.io/gh/TheOdinProject/curriculum/81a5d553f4073e593d23a6ab00d50eef8620796d/foundations/html_css/project/imgs/01.png), [Image Two (Color and Fonts)](https://cdn.statically.io/gh/TheOdinProject/curriculum/81a5d553f4073e593d23a6ab00d50eef8620796d/foundations/html_css/project/imgs/02.png)

If you click on the link for Image Two (Color and Fonts), then you are taken the the (outdated) image that shows the fonts as keywords instead of numeric values (i.e. as `extra-bold` instead of `900`).

The updated/new image can be found in `foundations/html_css/flexbox/project-landing-page/imgs/02.png`.
https://github.com/TheOdinProject/curriculum/blob/main/foundations/html_css/flexbox/project-landing-page/imgs/02.png

I would rather have a first-time contributor take this issue on. However, if I notice that no one volunteers, I will probably ask to be assigned after some time.

PR that fixed it last time: #25423 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-landing-page

### (Optional) Discord Name

Eduardo06sp

**Comment by MaoShizhong** [comment]
Oh, interesting...so the new image was added but the second PR needed to change the link never happened. Thanks for digging this up, @Eduardo06sp!

## Acceptance criteria

- [ ] The CDN link for the font design image is replaced with a new link to the fixed image. 
  - To do this, follow part 2 ["Create Statically links"](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum#create-statically-links) of our instructions on adding images to the curriculum.

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you've been assigned by a maintainer.

**Comment by dellaighted** [comment]
Hi, I would like to help!

**Comment by dellaighted** [comment]
> ## Acceptance criteria
> * [ ]  The CDN link for the font design image is replaced with a new link to the fixed image.
>   
>   * To do this, follow part 2 ["Create Statically links"](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum#create-statically-links) of our instructions on adding images to the curriculum.

I followed the acceptance criteria and fixed the second image. I ran the markdown lint on the project-landing-page.md file, and there was an error that pointed to line 15. I checked it, and it's a note instead of a subsection, so I'm not sure if you want that changed.
![image](https://github.com/user-attachments/assets/4e107eaf-9a12-4995-ab56-29295c02d2e2)
![image](https://github.com/user-attachments/assets/5874c33b-670f-4c4e-95a5-5ab9adfd7d9e)

**Comment by MaoShizhong** [comment]
Since the file you were working on was for a project, you needed to run the `lint:project` script, which uses the heading structure config for projects which differs from lessons.

All tests pass using the project config :+1:

**Comment by dellaighted** [comment]
Ah silly me! Good to hear the tests passed. 🚀
